### ISSUE TITLE
ci: rocm build is slow...

### ISSUE THREAD

**Issue by germasch** [wdmapp/gtensor#89]
So I guess I'm not quite sure why the builds with rocm are so slow in the first place...

But since the initialization / rocm install only takes of the order of 1 minute, it seems like it may be worth it to split this up in some fashion so that the various builds can run in parallel in separate containers (if possible without too much hassle).

**Comment by bd4** [comment]
Used to be only slightly slower than CUDA build, not sure why it changed. Perhaps the change in ROCm version, or that I fixed it so fft and blas are actually built. But that should slow down cuda build as well.
